CONTRIBUTING.rst

```diff
@@ -1,6 +1,11 @@
 Contribution guidelines
 =======================
 
+This guide describes rules for how to get your contributions into Anaconda. However, if you are seeking
+for help on how to implement changes to Anaconda please follow our
```

Comment:
information about implementing changes in Anaconda, please follow our
-or-
help with implementing changes in Anaconda, please follow our

I like the "help" word, but I can't bend it towards "the doc will help you get it" instead of "we'll help you do it".

Reply:
I used this solution:
``help with implementing changes in Anaconda, please follow our``
I guess I also like the "help" word :)